codecs/jxl/enc/jxl_enc.cpp

```diff
@@ -64,7 +64,7 @@ val encode(std::string image, int width, int height, JXLOptions options) {
     cparams.responsive = 0;
   }
 
-  io.metadata.SetAlphaBits(8);
+  io.metadata.m.SetAlphaBits(8);
```

Comment:
Hmm, what is `m.`? It's not very self-descriptive :)

Reply:
`metadata.m` is what used to be `metadata` - this is a bit of an intermediate state between refactors...